src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/stash/StashApiClient.java

```diff
@@ -201,6 +201,20 @@ private CloseableHttpClient getHttpClient() throws StashApiException {
     return httpClientBuilder.build();
   }
 
+  @Nonnull
+  private static String responseAsString(HttpResponse httpResponse) throws StashApiException {
```

Comment:
I would name it `responseEntirtyAsString` or `entityAsString` 

Current name is a misnomer, since it's a string representation of one part of response, not whole of it

Reply:
Renamed to `entityAsString`.